Tests/test_file_dds.py

```diff
@@ -170,7 +171,8 @@ def test_short_header():
         img_file = f.read()
 
     def short_header():
-        Image.open(BytesIO(img_file[:119]))
+        with Image.open(BytesIO(img_file[:119])):
+            pass
```

Comment:
let's stick a directive to skip coverage on this line, it's intentionally unreachable (from an exception in `Image.open`)